.github/workflows/release.yml

```diff
@@ -135,7 +135,9 @@ jobs:
           git commit -m "Update to version ${{ github.ref_name }}"
           git show
           git push git@github.com:cockpit-project/org.cockpit_project.CockpitClient HEAD
-          gh auth login
+
+          echo {{ GITHUB_TOKEN }} | gh auth login --with-token
```

Comment:
From the docs:

> Run [gh auth login](https://cli.github.com/manual/gh_auth_login) to authenticate with your GitHub account. Alternatively, gh will respect the GITHUB_TOKEN [environment variable](https://cli.github.com/manual/gh_help_environment).

Reply:
Yeah. Though it's a mess in GitHub workflows tbh https://github.com/cli/cli/issues/2922